doc/solver/solver_main.rst

```diff
@@ -0,0 +1,44 @@
+Solver architecture
+=========================
+To solve one time step of the simulation the solver is used.
+This solver is solving both the hydraulic and thermo dynamic equations.
+As unknown the following quantities are solved:
+
+#. Mass flow rate [kg/s]
+#. Total pressure [Pa]
+#. Specific internal energy [J/kg]
+
+The later is depending on the temperature and specifies the amount of energy needed to change the
```

Comment:
Typo :  the **latter** is depending on....

Reply:
fixed